aten/src/ATen/native/cuda/CUDALoops.cuh

```diff
@@ -150,7 +150,8 @@ static inline void launch_vectorized_kernel(
   int64_t grid = (N + io_block_work_size<io_size>() - 1) / io_block_work_size<io_size>();
   auto stream = at::cuda::getCurrentCUDAStream();
   using cpp_type = typename function_traits<func_t>::result_type;
-  const uint16_t max_vec_size = memory::can_vectorize_up_to<func_t>(data, sizeof(cpp_type) == 2);
+  // Here 1-byte dtypes are intentionally avoided as the vec8 causes incorrect outputs
```

Comment:
Sorry what? Why does vec8 cause incorrect outputs? It has to be fixed. some 1-byte ops already use 8 elements per thread, so why would 8-vectorization be different?